app/helpers/conditions_helper.rb

```diff
@@ -93,77 +99,88 @@ def section_info(plan, section)
     section_hash
   end
 
-
-	# returns a collection of questions to remove (hide). Choose from these which are to be removed by a given condition
+  # TODO: the logic of this is WAY too complex
```

Comment:
thanks for adding this. I have added some `TODO` comments in other places as well. We should do a search and create issues for any of them at some point